cmd/cmdtest/test_cmd.go

```diff
@@ -79,7 +79,7 @@ func (tt *TestCmd) Run(name string, args ...string) {
 // InputLine writes the given text to the childs stdin.
 // This method can also be called from an expect template, e.g.:
 //
-//     cli.expect(`Passphrase: {{.InputLine "password"}}`)
```

Comment:
This file seems to be modified by auto-formatting tool. I would suggest remove files which are unrelated to this change.